### ISSUE TITLE
Bad URL Regex

### ISSUE THREAD

**Issue by plasmatech8** [soufianesakhi/youtube-playlist-helper#5]
Minor issue.

The regular expression incorrectly matches YouTube URLs which are not under the '/watch' resource.

If it is not a '/watch' URL, it will attempt to insert the wrong part of the URL as the video ID and the URL is broken.

i.e. Bookmarks:
* https://www.youtube.com/feed/subscriptions - matches with 'subscriptions' as group 1
* https://www.youtube.com/c/NightblueMusic/videos - matches with 'videos' as group 1
* https://www.youtube.com/feed/trending - matches with 'trending' as group 1

Might result in something like:
```
https://www.youtube.com/watch_videos?video_ids=
qSWNhKJU3VM,
h6MUWEPvYHM,
subscriptions,          <-- bad
videos,                 <-- bad
trending,               <-- bad
Gl6ywXS4COs,
3rgw7fxNRgw,
```

Is it suitable to remove the second `?` from [(?:watch|embed)?](https://github.com/soufianesakhi/youtube-playlist-helper/blob/4243d061b9eacc72f59ff8f1d49fb8908edf3590/src/popup/popup.js#L172) (https://regex101.com/r/GYze1n/2)

**Comment by plasmatech8** [comment]
Recurse subfolders and shuffle playlist are some ideas.

**Comment by plasmatech8** [comment]
Fyi, I made a [fork](https://github.com/plasmatech8/youtube-playlist-helper) which is intended for web-dev practice, but can be pulled if desired. It contains the regex change and a 'shuffle playlist' setting.

**Comment by soufianesakhi** [comment]
Hello thanks for the contribution, unfortunatly the regex doesn't match these type of urls:
- http://youtu.be/iwGFalTRHDA
- www.youtu.be/iwGFalTRHDA
- youtu.be/iwGFalTRHDA

**Comment by soufianesakhi** [comment]
Fixed in  v1.2.1. Can you make a pull request if you have some contributions you want to make ? thanks
